internal/cache/controller_obj_cache.go

```diff
@@ -101,12 +101,12 @@ func (c *AviObjCache) AviRefreshObjectCache(client []*clients.AviClient, cloud s
 	c.PopulatePkiProfilesToCache(client[0], tenant)
 	c.PopulatePoolsToCache(client[1], cloud, tenant)
 	c.PopulatePgDataToCache(client[2], cloud, tenant)
+	c.PopulateStringGroupDataToCache(client[8], cloud)
```

Comment:
is client[8] not throwing index out of bound error ?

Reply:
We create Avi Rest Client pool of size 9 so it does not.